### DISCUSSION TITLE
Issue on [@types/node-replicate]

### DISCUSSION THREAD

**Discussion by manelcomiche** [DefinitelyTyped/DefinitelyTyped#65098] [Issues with a @types package]
The package changed completely, the type is invalid.

[a comment by an automated account is not shown]

**Comment by Ankan002** [comment]
Actually I recently noticed that and would try to update the types soon.

**Comment by Ankan002** [comment]
Meanwhile you might try to contribute if you can infer the types... it would be really helpful...😊😊
